### DISCUSSION TITLE
2.x loras

### DISCUSSION THREAD

**Discussion by adso3210** [zero01101/openOutpaint#235] [Q&A] [answered]
hello, I have a question: is there any way to use loras trained on 2.x with this extension?
or maybe someone knows how to train loras with 2.x that are compatible with the standard auto1111 syntax <lora:filename:multiplier> (I noticed that there are loras with these properties in civitai but I don't know how to make them.)
I use kohya_ss, but the 2.x loras it gives me can only be activated from the "additional networks" extension which is not present in the openoutpaint tab. Maybe I'm missing something.
Thanks!

**Comment by zero01101** [comment, marked as answer]
hey, sorry for the delay; the lora support is really remedial and just lists what A1111's API returns as an available lora; honestly, i didn't even realize loras trained against SD2.x were a thing at all :|  i'm not sure why they wouldn't be listed in webUI's native responses, but yeah, if they don't show up in additional networks tab (native in webUI >= 1.5 i think? not the abandoned(?) kohya extension) then they wouldn't populate in openOutpaint's lora chooser... you can always just simply type the lora params into the prompt manually however, that should work :)

**Reply by adso3210** [reply]
Thanks for replying! So much happened since sd2.x, but I still use the extension, it is the most convenient in my opinion.
Cheers!
